### ISSUE TITLE
Not working with ng-cli and angular4

### ISSUE THREAD

**Issue by saulide** [aminpaks/bound-sensor#1]
I get error ERROR in BoundSensorModule is not an NgModule, any ideas?

similar to: https://github.com/angular/angular-cli/issues/3157 ?

**Comment by aminpaks** [comment]
I will look into this issue. In meanwhile you can import the `Directive` instead of using the `NgModule` like this:
```
import { BoundSensorDirective } from 'bound-sensor';

@NgModule({
    declarations: [
        (...)
        BoundSensorDirective,
    ],
})
export class MyHomeModule { }
```

**Comment by aminpaks** [comment]
Just pushed a fix `bound-sensor@0.1.5`. Now it should be okay.

**Comment by aminpaks** [comment]
I'm close this because I believe the issue should be fixed.
